<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-US5V72R" ignored="0" name="BankingApp Test" tests="7" failures="5" timestamp="2021-07-28T15:35:32 EDT" time="74.17" errors="0">
  <testcase name="loginTest" time="1.295" classname="com.inetbanking.testCases.TC_LoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:38)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginDDT" time="7.75" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="4.027" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="3.949" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="7.344" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="4.022" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="addNewCustomer" time="14.227" classname="com.inetbanking.testCases.TC_AddCustomerTest_003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_AddCustomerTest_003.addNewCustomer(TC_AddCustomerTest_003.java:63)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addNewCustomer -->
</testsuite> <!-- BankingApp Test -->
